functions.go

```diff
@@ -10,6 +13,33 @@ type Function struct {
 	Return     Type
 }
 
+// IsImplementedForSyscall reports whether the function is implemented for syscall or not.
+func (f Function) IsImplementedForSyscall() bool {
+	// TODO: Use syscall.Syscall18 when Go 1.12 is the minimum supported version.
+	if len(f.Parameters) > 15 {
+		return false
+	}
+	return true
+}
+
+// Syscall returns a syscall expression for Windows.
+func (f Function) Syscall() string {
```

Comment:
I wonder if this would be more idiomatic/clear if it returned an error when !IsImplementedForSyscall (rather than having a separate func that needs to be called)?

Reply:
This is called at templates to generate `Syscall` calling, then returning error here would not make sense.